ipa-core/src/query/runner/oprf_ipa.rs

```diff
@@ -113,11 +113,11 @@ where
 
         let aws = config.attribution_window_seconds;
         match config.per_user_credit_cap {
-            8 => oprf_ipa::<C, BA8, BA3, BA20, BA3, F>(ctx, input, aws).await,
-            16 => oprf_ipa::<C, BA8, BA3, BA20, BA4, F>(ctx, input, aws).await,
-            32 => oprf_ipa::<C, BA8, BA3, BA20, BA5, F>(ctx, input, aws).await,
-            64 => oprf_ipa::<C, BA8, BA3, BA20, BA6, F>(ctx, input, aws).await,
-            128 => oprf_ipa::<C, BA8, BA3, BA20, BA7, F>(ctx, input, aws).await,
+            8 => oprf_ipa::<C, BA8, BA3, HV, BA20, BA3, 256>(ctx, input, aws).await,
```

Comment:
is it worth to have a generic histogram value parameter with everything else being hardcoded or I misunderstand what's going on here?

Reply:
I think that's a fair question -- I was just modeling it after what was already there (`HV` is the bitwise equivalent of `F`).

Reply:
Yeah, this is an unsolved problem. We never made anything runtime configurable. We just re-built the code for every different combination of query params =).